platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/Core.swift

```diff
@@ -0,0 +1,85 @@
+import PolyPodCore
+import Foundation
+import FlatBuffers
+
+/// Possible errors that can be thrown by PolyPodCoreSwift
+public enum PolyPodCoreError: Error {
+    /// Internal Rust Core failure with error code and message
+    case internalCoreFailure(Failure)
+    /// Rust Core returned an invalid result type for a given operation
+    case invalidResult(String)
+    /// Rust Core returned an invalid failure content
+    case invalidFailure(String)
+    
+    var localizedDescription: String {
+        switch self {
+        case let .internalCoreFailure(content):
+            return "Internal Core Failure: \(content.code) \(String(describing: content.message))"
+        case let .invalidResult(content):
+            return "Core Failure: \(content)"
+        case let .invalidFailure(content):
+            return "Core Failure: \(content)"
+        }
+    }
+}
+
+/// Swift wrapper around the Rust Core. Encapsulates specific Pointer and Flabuffer operations.
+public final class Core {
+    public static let instance = Core()
+    
+    // MARK: - Private config
+    private var languageCode: UnsafePointer<CChar>!
```

Comment:
Can you elaborate? I don't quite understand what you mean by `how to generate this documentation`.

Reply:
Aren't these docstrings for the documentation? At least the triple slash above are, right?

Reply:
`///` is used for documentation. `// MARK: -` is used to separate sections of the code, also facilitates navigation, this will be displayed by Xcode:
<img width="283" alt="Screenshot 2022-03-29 at 15 07 08" src="https://user-images.githubusercontent.com/12482662/160607634-b9d131ab-c980-4b70-80a2-12762f072d94.png">

Reply:
I didn't know that :-)
I guess documentation is a good thing anyway, but those docstrings were probably there before you started that PR and is outside the scope.